Makefile

```diff
@@ -42,7 +42,13 @@ integration-test: image ## Perform integration test
 ###############
 .PHONY: image
 image: ## Build docker image for the project
-	DOCKER_BUILDKIT=1 docker build -t $(REPO):latest .
+	DOCKER_BUILDKIT=1 docker build -t $(REPO):$(TAG) .
+
+.PHONY: docker-push
+docker-push: ## push to any docker registry
+	make image
+	docker push $(REPO):$(TAG)
```

Comment:
I'm a little concerned about recursively invoking `make`... I know this can be fragile.

Can we accomplish the same thing with:
```suggestion
# push to any docker registry
.PHONY: docker-push
docker-push: image
	docker push $(REPO):$(TAG)
```